addons/hr_skills/static/src/fields/resume_one2many/resume_one2many.js

```diff
@@ -30,7 +32,34 @@ ResumeListRenderer.rowsTemplate = "hr_skills.ResumeListRenderer.Rows";
 ResumeListRenderer.recordRowTemplate = "hr_skills.ResumeListRenderer.RecordRow";
 
 
-export class ResumeX2ManyField extends SkillsX2ManyField {}
+export class ResumeX2ManyField extends SkillsX2ManyField {
+    // All this overriding is to change the title of the modal
+    setup() {
+        super.setup();
+        const { saveRecord, updateRecord } = useX2ManyCrud(
+            () => this.list,
+            this.isMany2Many
+        );
+
+        const openRecord = useOpenX2ManyRecord({
+            resModel: this.list.resModel,
+            activeField: this.activeField,
+            activeActions: this.activeActions,
+            getList: () => this.list,
+            saveRecord: async (record) => {
+                await saveRecord(record);
+                await this.props.record.save();
+            },
+            updateRecord: updateRecord,
+            withParentId: this.props.widget !== "many2many",
+        });
+
+        this._openRecord = (params) => {
+            params.title = _t("Create a resume line");
+            openRecord(params);
+        };
+    }
```

Comment:
I think a cleaner way of doing this to avoid code duplication would be to :
- In skills_one2many.js define a function getTitleName() and use that in the original setup
- In resume_one2many.js, just override that getTitleName()

Reply:
Yes thank you 🙏